File: Application/src/main/java/com/shareyourproxy/app/fragment/AboutFragment.java
package com.shareyourproxy.app.fragment;

import android.annotation.SuppressLint;
import android.os.Bundle;
import android.support.v4.app.Fragment;
import android.support.v7.widget.Toolbar;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.shareyourproxy.R;
import com.shareyourproxy.app.AboutActivity;

import butterknife.Bind;
import butterknife.ButterKnife;

/**
 * Show an Apache II License for this project.
 */
public class AboutFragment extends BaseFragment {
    @Bind(R.id.fragment_about_toolbar)
    Toolbar toolbar;

    /**
     * Return a new instance of this fragment for the parent {@link AboutActivity}.
     *
     * @return AboutFragment
     */
    public static Fragment newInstance() {
        return new AboutFragment();
    }

    @Override
    @SuppressLint("InflateParams")
    public View onCreateView(
        LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        View rootView = inflater.inflate(R.layout.fragment_about, null, false);
        ButterKnife.bind(this, rootView);
        buildToolbar(toolbar, getString(R.string.about), null);
        return rootView;
    }

}

File: Application/src/main/kotlin/com/shareyourproxy/app/fragment/AboutFragment.kt
package com.shareyourproxy.app.fragment

import android.os.Bundle
import android.support.v7.widget.Toolbar
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import com.shareyourproxy.R
import com.shareyourproxy.R.layout.fragment_about
import com.shareyourproxy.R.string.about
import com.shareyourproxy.util.ButterKnife.bindString
import com.shareyourproxy.util.ButterKnife.bindView

/**
 * Show an Apache II License for this project.
 */
internal final class AboutFragment() : BaseFragment() {
    private val toolbar: Toolbar by bindView(R.id.fragment_about_toolbar)
    private val title: String by bindString(about)

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View {
        return inflater.inflate(fragment_about, null, false);
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        buildToolbar(toolbar, title)
    }
}
